scripts/tests/test_validate_docstrings.py

```diff
@@ -1052,6 +1052,12 @@ def test_raises_for_invalid_attribute_name(self, invalid_name):
         with pytest.raises(AttributeError, match=msg):
             validate_docstrings.Docstring(invalid_name)
 
+    @pytest.mark.parametrize('name', [
+        'pandas.Series.str.isdecimal', 'pandas.Series.str.islower'])
+    def test_encode_content_write_to_file(self, name):  # GH25466
```

Comment:
This comment should be on the next line. @kpapdac if you want to fix up otherwise can clean up before merging

Reply:
yeah, I'm sorry, you said that above but I didn't get it..I'll fix it.